smarts/core/smarts.py

```diff
@@ -954,7 +954,7 @@ def providers(self) -> List[Provider]:
         # TODO: It's inconsistent that pybullet is not here
         return self._providers
 
-    def get_provider_by_type(self, requested_type) -> Provider:
+    def get_provider_by_type(self, requested_type):
```

Comment:
@Gamenot Can you look over these changes in `smarts.py`? This was to fix pytype errors when it would return `None` instead of `Provider`